### PR TITLE
feat: treemap, heatmap series theme

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

```ts
interface TreemapChartSeriesTheme {
  colors?: string[];
  startColor?: string;
  endColor?: string;
  border?: BorderTheme;
  select?: {
    border?: BorderTheme;
    color?: string;
  };
  hover?: {
    border?: BorderTheme;
    color?: string;
  };
}
```

image

image

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/component/heatmapSeries.ts [hunk 1]
```diff
@@ -103,14 +106,29 @@ export default class HeatmapSeries extends Component {
           colorRatio,
           color: getSpectrumColor(colorRatio, distances, startRGB),
           thickness,
+          borderColor,
         };
       });
     });
   }
 
+  getRespondersWithTheme(responders: HeatmapRectResponderModel[], type: 'select' | 'hover') {
```
Comment (jungeun-cho) [inline src/component/heatmapSeries.ts:115]: `'select' | 'hover'`는 다른데서도 사용될것같은데, 타입 지정하고 사용하는건 어떻할까요?

**Review by jungeun-cho — COMMENTED** [review]
[11/03] 이견없습니다. 수고하셨어요~

트리맵, 히트맵 옵션에서 테마를 지정하지 않는 것은 디폴트 호버 스타일이 나타나지 않아요.
콘솔 로그 확인부탁드려요.
스크린샷 2020-11-03 01 10 18

**Review by shiren — APPROVED** [review]
수고하셨어요

File: src/component/heatmapSeries.ts (hunk 1 above)
Comment (jung-han) [reply src/component/heatmapSeries.ts:115]: `RespondersThemeType`로 처리하겠습니다. 이후에 사용하실때 이용하시면 될 것 같습니다 ㅎ

**Pull request merged**